forum/startups/tests.py

```diff
@@ -31,3 +34,233 @@ def test_industries_list(self):
+
+        self.assertEqual(
+            response.status_code,
+            status.HTTP_200_OK,
+            "Status code is different from 200"
+        )
+        self.assertEqual(
+            len(response.json()),
+            2,
+            "The number of objects received is not equal to the expected number of startups(one has is_deleted=True)"
+        )
+
+    def test_get_startups_empty_list(self):
+        response = self.client.get(reverse('users:user_startups', kwargs={'user_id': 6666}))
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK, "Status code is different from 200")
+
+        self.assertEqual(len(response.json()), 0, "The number of objects received is more than 0")
+
+    def test_create_startup(self):
+        response = self.client.post(
+            reverse('users:user_startups', kwargs={'user_id': self.test_user.user_id}),
+            {
+                'user': self.test_user.user_id,
+                'name': 'test name4',
+                'location': 'test location',
+                'description': 'test description',
+                'contacts': json.dumps({'phone': 'test phone'})
+            }
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED, "Status code is different from 201")
+
+        self.assertIn('startup_id', response.json())
+
+        startup = Startup.objects.get(startup_id=response.json()['startup_id'])
+
+        self.assertEqual(startup.name, 'test name4')
+
+    def test_create_startup_failed(self):
+        response = self.client.post(
+            reverse('users:user_startups', kwargs={'user_id': self.test_user.user_id}),
+            {'user': self.test_user.user_id, 'name': 'test name5'}
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST, "Status code is different from 400")
+
+
+class UserStartupTestCase(BaseUserStartupsTestCase):
+
+    def test_get_startup(self):
+        response = self.client.get(
+            reverse(
+                'users:user_startup',
+                kwargs={'user_id': self.test_user.user_id, 'startup_id': self.startup3.startup_id}
+            )
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK, "Status code is different from 200")
+
+        self.assertTrue(isinstance(response.json(), dict))
+
+        self.assertIn('startup_id', response.json())
+        self.assertEqual(response.json()['startup_id'], self.startup3.startup_id, "startup_id is not correct")
+
+        self.assertIn('size', response.json())
+        self.assertTrue(isinstance(response.json()['size'], dict))
+        self.assertEqual(len(response.json()['size']), 4, "Length of size object is not correct")
+
+        self.assertIn('project', response.json())
+        self.assertTrue(isinstance(response.json()['project'], dict))
+        self.assertListEqual(
+            list(response.json()['project'].keys()), ['title', 'last_updated'], "Project keys are bad"
+        )
```

Comment:
Ensure all expected fields are validated

Reply:
These are all the expected fields. This is checked by self.assertListEqual().